pkg/proc/variables.go

```diff
@@ -614,12 +620,38 @@ func (scope *EvalScope) SetVariable(name, value string) error {
 
 // LocalVariables returns all local variables from the current function scope.
 func (scope *EvalScope) LocalVariables(cfg LoadConfig) ([]*Variable, error) {
-	return scope.variablesByTag(dwarf.TagVariable, &cfg)
+	vars, err := scope.Locals()
+	if err != nil {
+		return nil, err
+	}
+	vars = filterVariables(vars, func(v *Variable) bool {
+		return (v.Flags & (VariableArgument | VariableReturnArgument)) == 0
+	})
+	loadValues(vars, cfg)
+	return vars, nil
 }
 
 // FunctionArguments returns the name, value, and type of all current function arguments.
 func (scope *EvalScope) FunctionArguments(cfg LoadConfig) ([]*Variable, error) {
-	return scope.variablesByTag(dwarf.TagFormalParameter, &cfg)
+	vars, err := scope.Locals()
+	if err != nil {
+		return nil, err
+	}
+	vars = filterVariables(vars, func(v *Variable) bool {
+		return (v.Flags & (VariableArgument | VariableReturnArgument)) != 0
```

Comment:
I think we should exclude return args here since we can distinguish now and have a new method that is `FunctionReturnVals`.

Reply:
That wouldn't be backwards compatible for the API (unless we concatenate this and FunctionReturnVals in debugger.go or rpc2/server)

Reply:
That's true, however unfortunate.